src/features/canvass/components/CanvassSelectAreaPage.tsx

```diff
@@ -0,0 +1,144 @@
+'use client';
+
+import { FC, useState } from 'react';
+import React from 'react';
+import { ArrowForwardIos } from '@mui/icons-material';
+import {
+  Avatar,
+  Box,
+  CircularProgress,
+  Divider,
+  List,
+  ListItem,
+  ListItemButton,
+  Typography,
+} from '@mui/material';
+import { useRouter } from 'next/navigation';
+
+import useMyCanvassAssignments from '../hooks/useMyAreaAssignments';
+import { ZetkinAreaAssignment } from '../../areaAssignments/types';
+import useOrganization from 'features/organizations/hooks/useOrganization';
+import ZUIFutures from 'zui/ZUIFutures';
+import oldTheme from 'theme';
+import { Msg } from 'core/i18n';
+import messageIds from '../l10n/messageIds';
+import useAssignmentAreas from 'features/areaAssignments/hooks/useAssignmentAreas';
+
+const Page: FC<{
+  assignment: ZetkinAreaAssignment;
+}> = ({ assignment }) => {
+  const orgFuture = useOrganization(assignment.organization_id);
+  const router = useRouter();
+  const areas = useAssignmentAreas(assignment.organization_id, assignment.id);
+  const [loadingAreaId, setLoadingAreaId] = useState<number | null>(null);
+  return (
+    <ZUIFutures futures={{ org: orgFuture }}>
+      {({ data: { org } }) => (
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            height: '100dvh',
+            width: '100vw',
+          }}
+        >
+          <Box
+            alignItems="center"
+            bgcolor={oldTheme.palette.background.paper}
+            display="flex"
+            justifyContent="space-between"
+            padding={2}
+          >
+            <Box>
+              <Typography variant="body1">{assignment.title}</Typography>
+              <Box alignItems="center" display="flex" gap={1}>
+                <Avatar
+                  src={`/api/orgs/${org.id}/avatar`}
+                  sx={{ height: 24, width: 24 }}
+                />
+                <Typography variant="body2">{org.title}</Typography>
+              </Box>
+            </Box>
+          </Box>
+          <Divider />
+          <Box>
+            {areas.length > 0 ? (
+              <List disablePadding>
+                {areas.map((area) => (
+                  <React.Fragment key={area.id}>
+                    <ListItem key={area.id} disablePadding>
+                      <ListItemButton
+                        onClick={async () => {
+                          setLoadingAreaId(area.id);
+                          await router.push(
+                            `/canvass/${assignment.id}/area/${area.id}`
+                          );
```

Comment:
I think we should generally avoid using `router.push()` in cases where we can use `href` instead. That way, the browser will understand that something is a link, and can give the user option to open in separate tabs etc.

In this case, I expect `ListItemButton` to have a `href` property (as is often the case in MUI) but I could be wrong.

Reply:
It has a `href` property and it's working good. But I'm curious about why don't use the <Link> component that NextJS has?